### PR TITLE
Fix cere types, udpate configs

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2077

### CODE REVIEW

File: api/.env.default [hunk 1]
```diff
@@ -1,2 +1,8 @@
...
...olygon-relayer-3","minBalance":5,"group":"BRIDGE_RELAYERS","type":"NATIVE","tokenSymbol":"MATIC"},{"address":"0x8fe028Eb002bbc3ec45c5dF8acfFf67eC95B6f88","name":"polygon-erc20-handler","minBalance":10000000,"type":"ERC20","erc20TokenAddress":"0x2da719db753dfa10a62e140f436e1d67f2ddb0d6","tokenSymbol":"CERE","group":"BRIDGE"}]}]},{"name":"ETHEREUM","nativeTokenSymbol":"ETH","networks":[{"name":"MAINNET","rpcUrl":"https://mainnet.infura.io/v3/051b40aebf864545a7df3869dfa87e7b","cereTokenContractAddress":"0x2da719db753dfa10a62e140f436e1d67f2ddb0d6","accounts":[{"address":"0x8fe028Eb002bbc3ec45c5dF8acfFf67eC95B6f88","name":"ethereum-erc20-handler","minBalance":10000000,"type":"ERC20","erc20TokenAddress":"0x2da719db753dfa10a62e140f436e1d67f2ddb0d6","tokenSymbol":"CERE","group":"BRIDGE"}]},{"name":"TESTNET","cereTokenContractAddress":"0x0b10e304088b2BA2B2acfD2f72573FAaD31a13A5","rpcUrl":"https://goerli.infura.io/v3/178b87e860404572bc217ca53fe594bd","accounts":[{"address":"0x92c1576845703089CF6c0788379ED81f75F45dd5","name":"ethereum-erc20-handler","minBalance":10000000,"type":"ERC20","erc20TokenAddress":"0x0b10e304088b2BA2B2acfD2f72573FAaD31a13A5","tokenSymbol":"CERE","group":"BRIDGE"}]}]}]
```
Comment (AndreiNavoichyk) [inline api/.env.default]: Replace all Infura providers with public

**Review by AndreiNavoichyk — COMMENTED** [review]
@Dmitriy1984Molchanov 
1. Please, add to logs to which Cere Blockchain URLs API is connecting 
2. Update a CHANGELOG

File: api/.env.default (hunk 1 above)
Comment (DmitriyMolch) [reply api/.env.default]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 - Updated Devnet url
 - [FE] Updated link to Cere Docs 
+- Fix CERE types according to a new version of "@polkadot/api@^5.9.1"
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md]: Use perfective verb

**Review by AndreiNavoichyk — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (DmitriyMolch) [reply CHANGELOG.md]: fixed

**Pull request merged**